kotlin-detect-config.yaml

```diff
@@ -104,6 +106,7 @@ style:
             - "**/examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkShortCommand.kt"
             - "**/examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkVendorCommand.kt"
             - "**/examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/UnpairCommand.kt"
+            - "**/src/controller/java/generated/java/**/*"
```

Comment:
we can make jinja files output a newline at the end of the file

Reply:
I removed.